app/views/dashboards/show.html.erb

```diff
@@ -1,6 +1,13 @@
 <h1><%= @environment.name %></h1>
 
 <section>
+  <p>
+    <%= form_tag "", method: :get do %>
+      Status at <%= text_field_tag :before, @before.to_s(:db) %>
```

Comment:
If you use 'datetime_local_field_tag' you'll get a datepicker for free on most browsers.
![samson](https://cloud.githubusercontent.com/assets/8860832/6597428/8493868c-c7f4-11e4-9d78-e5173fa1b0be.png)